docs/userguides/networks.md

```diff
@@ -0,0 +1,70 @@
+# Network
+
+When interacting with the blockchain, you will have to select a network.
+
+## Selecting a Network
+
+Commonly, you will use the `--network` option to configre your network during Ape commands.
+The following is a list of common Ape commands that can use the `--network` option:
+
+```bash
+ape test --network ethereum:local:foundry
+ape run deploy --network ethereum:mainnet:geth  # NOTE: Must be defined if using CLI script
```

Comment:
Maybe some more clarity on what must be defined

Reply:
Done! Had to add the scripts guide to explain more